index.html

```diff
@@ -0,0 +1,141 @@
+               </ul>
+         <p>A daily selection of privately
+            personalized reads; no accounts 
+            or sign-ups required.</p>   
+         <button class="btn_3"> html</button>
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+         </section>
+         <button class="btn_2">See Project</button>
+     </div>
+
+     <div class="container">
+        <div> <img src="images/Portfolio_4.png" class="img_2"> </div>
+        <section class="section_1">
+         <h1> Multi-Post <br> Stories </h1>
+         <ul>
+           <li class="list_1"> CANOPY </li>
+           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
+           <li class="list3"><img src="images/Counter_2.png">2015</li>
+               </ul>
+         <p>A daily selection of privately
+            personalized reads; no accounts 
+            or sign-ups required.</p>   
+         <button class="btn_3"> html</button>
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+        </section>
+        <button class="btn_2">See Project</button>
+     </div>
+
+     <div class="container_2">
+        <section class = "section_2">
+          <h2> About <br> Myself</h2>
+          <p> I’m a software developer! I can help you build a product,
+            feature or website Look through some of my
+            work and experience! If you like what you see and
+             have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <a href="#" id="connect_1"> LET'S CONNECT</a>
+          <ul class="social_2">
+            <li><a href="https://github.com/david30-maker"><img src="images/github1.png" alt="github logo"></a></li>
+            <li><a href="https://www.linkedin.com/in/david-igbo-464196184"><img src="images/Linkedin_1.png" alt="in logo"></a></li>
+            <li><a href="https://venture.angellist.com/v/begin"><img src="images/angellist.png" alt="angel logo"></a></li>
+            <li><a href="https://twitter.com/@okpahdavid"><img src="images/twitter_1.png" alt="Twiiter logo"></a></li>
+            <li><a href="#"><img src="images/medium.png" alt="m logo"></a></li>
+            
+          </ul>
+        </section>
+    
+        <button class="btn"> Get my resume </button>
+
+        <div class="div-7">
+            <h2> Languages</h2>
+              <a href="#">
+              <i class="fa fa-angle-down"></i></a>
+          </div>
+
+          <div class="div-1">
+          JavaScript
+        </div>
+
+        
+        <div class="div-2">
+          HTML
+        </div>
+        
+        <div class="div-3">
+        CSS
+        </div>
+      
+          <div class="div-5">
+            <h3>Frameworks</h3>
+            <a href="#">❯</a>
+          </div> 
+          <hr>
+
+          <div class="div-6">
+            <h3>Skills</h3>
+            <a href="#">❯</a>
+          </div>
```

Comment:
- [ ] I see that your project in this section is also a bit different from the Figma design. The accordion is not well positioned as seen below. Kindly fix this.
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, and each of them in a `li`. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`. essentially, you will have top-level `ul` with three `li` and within each of the tree `li`, you will have `ul` nested within and with `li`.

Your design | Figma Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/213212609-b3a022c9-6e09-4e92-8357-5cd3f33e4045.png) | ![image](https://user-images.githubusercontent.com/84156714/213213207-f9e93d0f-1ac0-4c09-85ad-1522802d2bcd.png)